app/models/concerns/spree/gift_cards/order_concerns.rb

```diff
@@ -36,6 +36,14 @@ def send_gift_card_emails
           end
         end
       end
+
+      def non_gift_card_items
+        line_items.reject { |li| li.gift_card? }
+      end
+
+      def amount_without_gift_cards
+        non_gift_card_items.map(&:amount).sum
```

Comment:
Nice use of sum here over the painfully unperformant `.inject(:+)`. For some reason I still have muscle memory for the inject.